tests/llm/fixtures/test_ask_holmes/117_new_relic_tracing/setup.sh

```
@@ -1,6 +1,6 @@
-kubectl create namespace app-117
+kubectl apply -f instrumentation.yaml -n newrelic
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)